pg_keeper.c

```diff
@@ -327,11 +327,15 @@ execSQL(const char *conninfo, const char *sql)
 		ereport(LOG,
 				(errmsg("could not get tuple from server : \"%s\"",
 					conninfo)));
-
+		
+                PQclear(res); /* add */
```

Comment:
I vote for removing this comment here and line 337 since it is clear enough without it.
Also please use tabs instead of spaces for indent here for consistency with other code.

Reply:
+1 for removing.
Could you please update the commit? or shall I?

Reply:
申し訳ないです。
自由に変更してください。
よろしくお願いします。